src/Google.Api.Gax/FixedSizePage.cs

```diff
@@ -19,7 +19,14 @@ public sealed class FixedSizePage<TResource> : IEnumerable<TResource>
     {
```

Comment:
Yes, this is definitely something to think about. I'm not even sure we should be using Page for this, given that it's not necessarily a natural page from the API perspective.